### PR TITLE
Login and protected routes in backend

### PR DESCRIPTION
**Pull request by SajaLahaleeh**

### CODE REVIEW

File: client/src/components/ShardComponents/Login/style.css [hunk 1]
```diff
@@ -0,0 +1,30 @@
+body {
+  text-align: center;
+}
+
+button {
```
Comment (FarahZaqout) [inline client/src/components/ShardComponents/Login/style.css]: use more specific class names.

File: client/src/components/ShardComponents/Login/index.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        <label htmlFor="email"></label>
+        <input
+          placeholder="Email"
+          type="email"
+          id="email"
+          name="email"
+          value={this.state.email}
+          onChange={this.handelChange}
+        />
+        <br />
```
Comment (FarahZaqout) [inline client/src/components/ShardComponents/Login/index.js]: don't use ```<br />``` to add spaces between elements. This is designed to add line breaks for texts.

use margins instead.

File: client/src/components/ShardComponents/Login/index.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  }
+
+  render() {
+    return (
+      <form onSubmit={this.submit}>
+        <label htmlFor="email"></label>
+        <input
+          placeholder="Email"
+          type="email"
+          id="email"
```
Comment (FarahZaqout) [inline client/src/components/ShardComponents/Login/index.js]: u don't need to use id all the time. If you are not using the id for anything, don't add it.

Comment (jema28) [reply client/src/components/ShardComponents/Login/index.js]: She's using it to link to label here so no worries 🐼

File: client/src/components/SharedComponents/Login/validatedLoginForm.js [hunk 4]
```diff
@@ -0,0 +1,72 @@
+import React from 'react'
+import { Formik } from 'formik'
+import * as EmailValidator from 'email-validator'
+// import * as Yup from 'yup'
```
Comment (jema28) [inline client/src/components/SharedComponents/Login/validatedLoginForm.js]: Please delete this.

File: client/src/components/SharedComponents/Login/validatedLoginForm.js [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+
+const validatedLoginForm = () => {
+  console.log('heeeere')
```
Comment (jema28) [inline client/src/components/SharedComponents/Login/validatedLoginForm.js]: Please delete this.

File: client/src/components/SharedComponents/Login/validatedLoginForm.js [hunk 6]
```diff
@@ -0,0 +1,72 @@
+import React from 'react'
+import { Formik } from 'formik'
+import * as EmailValidator from 'email-validator'
+// import * as Yup from 'yup'
+
+const validatedLoginForm = () => {
```
Comment (jema28) [inline client/src/components/SharedComponents/Login/validatedLoginForm.js]: You can simplify this using an implicit return. 
```
() => ( 
..code... 
)
```

File: client/src/components/SharedComponents/Login/validatedLoginForm.js (hunk 4 above)
Comment (SajaLahaleeh) [reply client/src/components/SharedComponents/Login/validatedLoginForm.js]: I delete this file

**Pull request merged**